projects/noloan/common/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="hasadna.n.common" />
```

Comment:
What is `hasadna.n.common`?

Reply:
It suppose to be `hasadna.noloan.common`
fixing it now